Sources/CardPayments/APIRequests/SetupTokenDetailsRequest.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+#if canImport(CorePayments)
+import CorePayments
+#endif
+
+struct SetupTokenDetailsRequest: APIRequest {
```

Comment:
Requests using `GraphQLClient` don't need to conform to `APIRequest`

Reply:
See [this example](https://github.com/paypal/paypal-ios/blob/257df315bb8ab2e323df39bf17a5af48e36498a1/Sources/CorePayments/EligibilityAPI.swift#L36-L37) of how it's currently being done